tests/test_profiler.py

```diff
@@ -32,9 +22,9 @@ def test_simple_profiler():
 
     # different environments have different precision when it comes to time.sleep()
     # see: https://github.com/PyTorchLightning/pytorch-lightning/issues/796
-    _assert_almost_equal(p.recorded_durations["a"], [3, 1], tol=0.1)
-    _assert_almost_equal(p.recorded_durations["b"], [2], tol=0.1)
-    _assert_almost_equal(p.recorded_durations["c"], [1], tol=0.1)
+    np.testing.assert_allclose(p.recorded_durations["a"], [3, 1], atol=0.1)
```

Comment:
```
E       Max absolute difference: 0.150021
E       Max relative difference: 0.03750525
E        x: array(4.2)
E        y: array([4])
```

According to the build which failed, this `atol` is not large enough.

Reply:
Comes from CI tests...

Reply:
the `atol` for this line is 0.1, i think that 0.5 should be sufficient

Reply:
Sure =)